frontend/src/features/admin/access/ManageAccessRequests.test.tsx

```diff
@@ -70,11 +72,13 @@ const successStore = mockStore({
 
 const componentRender = (store: any) => {
   let component = create(
-    <Router history={history}>
-      <Provider store={store}>
-        <ManageAccessRequests />
-      </Provider>
-    </Router>,
+    <Formik initialValues={{}} onSubmit={noop}>
```

Comment:
due to some changes in the table, any test component wrapping the table needs to include a Formik provider to avoid warnings.